source/numerics.tex

```diff
@@ -1437,6 +1455,154 @@
 unless \tcode{T} is an unsigned integer type\iref{basic.fundamental}.
 \end{itemdescr}
 
+\rSec2[bit.rotate]{Rotating}
+
+In the following descriptions,
+let \tcode{N} denote \tcode{std::numeric_limits<T>::digits}.
```

Comment:
I don't think we need the `std::` qualification on `numeric_limits`, both here and other places below (sorry for not noticing that before we voted on it).

Reply:
Fixed.